### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by padrone1225**

### CODE REVIEW

**Review by padrone1225 — COMMENTED** [review]
OK

File: src/App.tsx [hunk 1]
```diff
@@ -1,78 +1,26 @@
...
 import { useEffect, useState } from "react";
 import "./App.css";
+import { TrafficLightBox } from "./components/TrafficLightBox";
 
 function App() {
-  // const colors = ["red", "yallow", "green"];
   const [colorType, setColorType] = useState(0);
-  let countTime = 0;
+  const TIME_INTERVAL = 1000;
+  const TIME_STEPS = [1, 2, 4];
```
Comment (hungry-engineer) [inline src/App.tsx:8]: Might be better to keep these vars outside of component body.

File: src/components/TrafficLightBox/TrafficLight/index.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import "./index.css";
+
+export const TrafficLight = ({
+  color,
+  isTurnOn,
+}: {
+  color: string;
+  isTurnOn: boolean;
```
Comment (hungry-engineer) [inline src/components/TrafficLightBox/TrafficLight/index.tsx:8]: I don't think this is a good name. isTurnOn doesn't seem like a good english word.

File: src/components/TrafficLightBox/index.css [hunk 3]
```diff
@@ -0,0 +1,9 @@
+.Traffic-box {
```
Comment (hungry-engineer) [inline src/components/TrafficLightBox/index.css:1]: Class name differs from component name. Traffic box and Traffic light box seems to be completely different name.

File: src/components/TrafficLightBox/index.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import "./index.css";
+import { LightColor } from "../../model/LightColor";
+
+export const TrafficLightBox = ({ lightType }: { lightType: number }) => {
+  const lightColors = [LightColor.Red, LightColor.Yellow, LightColor.Green];
+  return (
+    <div className="Traffic-box">
+      {lightColors.map((color, index) => (
+        <TrafficLight
+          key={index}
```
Comment (hungry-engineer) [inline src/components/TrafficLightBox/index.tsx:11]: key should be color, rather than index.

**Pull request merged**